hathor/websocket/factory.py

```diff
@@ -314,14 +332,42 @@ def _remove_connection_from_address_dict(self, connection: HathorAdminWebsocketP
         if len(self.address_connections[address]) == 0:
             del self.address_connections[address]
 
+    def _handle_subscribe(self, connection: HathorAdminWebsocketProtocol, message: Dict[Any, Any]) -> None:
+        """ Handler for subscription to a ws message."""
+        ws_message: Optional[str] = message.get('message')
```

Comment:
Isn't it easier to receive a list of strings?

Reply:
Yes, updated it! Can you see now?